docs/upgrade.md

```diff
@@ -10,20 +10,26 @@ This document describes the steps for upgrading the following OpenEBS reources:
 
 ## CSPC pools
 
-These instructions will guide you through the process of upgrading cStor CSPC pools from `1.10.0` or later to a newer release up to `2.4.0`.
+These instructions will guide you through the process of upgrading cStor CSPC pools from `1.10.0` or later to a newer release up to `2.5.0`.
 
 ### Prerequisites
 
 Before upgrading the pools make sure the following prerequisites are taken care of:
 
- - Upgrade the control plane components by applying the desired version of cstor-operator from the [charts](https://github.com/openebs/charts/tree/gh-pages). You can verify the current version of the control plane using the command:
+ - Upgrade the control plane components by applying the desired version of cstor-operator from the [charts](https://github.com/openebs/charts/tree/gh-pages). 
+ 
+ **Note:** If upgrading the control plane from 2.4.0 or previous to 2.5.0 or above version please clean up the CSIDriver CR before applying the operator using the below command.
+  ```sh
+  kubectl delete csidriver cstor.csi.openebs.io
+  ```
+ You can verify the current version of the control plane using the command:
     ```sh
     $ kubectl -n openebs get pods -l openebs.io/version=<version>
     ```
     where `<version>` is the desired version.
-    For example if desired version is `2.4.0` the output should look like:
+    For example if desired version is `2.5.0` the output should look like:
     ```sh
```

Comment:
n_: insert a line prior to this quote to help with format.

Reply:
Done